supabase/functions/alert-data-processing/index.ts

```diff
@@ -207,12 +152,42 @@ serve(async (_request: Request): Promise<Response> => {
         );
     }
 
-    await emailNotifications(pendingEmails, resendToken, senderAddress);
+    const responses = await emailNotifications(
+        pendingEmail,
+        resendToken,
+        senderAddress
+    );
+
+    const errors = responses.filter((response) => response.status >= 400);
+
+    if (errors.length > 0) {
+        console.info(`${responses.length - errors.length} emails sent.`);
```

Comment:
It might be nice to log the `catalog_name` here, to make it easier to correlate errors with catalog items. I could see also consolidating the separate logs into a single statement, like:

```
console.log("finished sending emails", {"catalog_name": catalog_name, "attempted": responses.length, "errors": errors });
```